### ISSUE TITLE
Merge npm and github records

### ISSUE THREAD

**Issue by redbar0n** [aantipov/moiva#489]
Sometimes, github repos are not named the same as their corresponding npm packages:

https://github.com/matthewp/robot
https://www.npmjs.com/package/robot3

This is not recognised by Moiva, which either doesn't find the npm package, or if the user finds it manually then Moiva will show its results separate:
https://moiva.io/?npm=robot3&github=matthewp/robot

The best would be for Moiva to automatically merge them, but that may be hard (scrape their github referenced website for npm links?).
The second best would be for the community to be able to manually merge records such as this.
The third best would be to have a button or similar so they can quickly be sent as a suggestions for merging. So that you don't have a github issue like this per library.

**Comment by aantipov** [comment]
Hi @redbar0n 

TLDR:
1. Users should use Moiva's NPM search to find and compare Npm packages. GitHub metrics and stats are always there as well.
2. Users should use Moiva's GitHub search to find and compare GitHub repos. There will be no NPM stats in most cases.

In 99% cases every npm package has a link to a GitHub repo. You can see that repo on the npm package page 
So when a user searches for an Npm package, Moiva knows what is the related GitHub repo and shows the data.

That doesn't work in the opposite direction, though. None of GitHub apis tell you what is the related npm package.
A GitHub repo can be a source of multiple different npm packages, php packages, maven packages at the same time.
Hence, there is no 1:1 Github -> Npm relation and Moiva doesn't make any assumption here.

That was the reason, why I made Npm a default search and repurposed Moiva as a tool to "Evaluate and Compare NPM packages" - users should use it to find and compare Npm packages. Otherwise they won't see NPM related stats.

Users can use "GitHub" search, but they should not expect any NPM related metrics and data because of the above reason. It's useful when you want to see some GitHub repo from a completely different ecosystem - Java, PHP, etc.

That being said, for libraries from Moiva's Catalog it's possible to derive NPM package from a Github repo and Moiva does it. Check, for example, Vue or React.


So in your particular case I suggest to use Npm search to find `robot3` package and you'll have all the data.

Sorry for the long text. I hope I made it more clear :)
I'm open to any suggestions here if you think current behaviour is confusing or anything.

**Comment by redbar0n** [comment]
Ah, ok, I see. Makes sense. Thanks for the candid reply. I see that it actually found the github repo from NPM for that package as well, actually.
